index.bs

```diff
@@ -449,7 +456,9 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
         member represents that the video track includes the specified HDR
         metadata type, which the UA needs to be capable of interpreting for tone
         mapping the HDR content to a color volume and luminance of the output
-        device. Valid inputs are defined by {{HdrMetadataType}}.
+        device. Valid inputs are defined by {{HdrMetadataType}}. hdrMetadataType is
+        only applicable to {{MediaDecodingConfiguration}} of the types {{media-source}}
```

Comment:
nit (here and later): s/of the types/for types/

Reply:
Thanks for the correction! It's sometimes hard to get this right since I'm not a native speaker :)

Reply:
Ha, I'm actually not sure if you're English was incorrect... may just be my personal style.